apps/www/pages/404.tsx

```diff
@@ -41,10 +41,9 @@ const Error404 = () => {
         </div>
         <div className="absolute">
           <h1
-            className={`select-none opacity-[5%] filter transition duration-200 ${
+            className={`text-scale-1200 select-none text-9xl opacity-[5%] filter transition duration-200 ${
```

Comment:
The 404 text size is to small on bigger screens, and cant see very well.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/70828596/179298047-754f5708-740f-4c07-8d17-52e0f87f2e05.png">

Reply:
@Isaiah-Hamilton - I've just made some changes now and should be looking better now:

<img width="1440" alt="Screenshot 2022-07-16 at 10 27 32" src="https://user-images.githubusercontent.com/22655069/179349200-4e233f29-676e-4a11-8e2c-b4211fa50d29.png">